tests/csv/grch37/sample_bam_grch37.csv

```diff
@@ -0,0 +1,2 @@
+sample,bam,bai
+NA12878,s3://gatk-test-data/wgs_bam/NA12878_20k_b37/NA12878.b37.bam,s3://gatk-test-data/wgs_bam/NA12878_20k_b37/NA12878.b37.bam.bai
```

Comment:
Why don't you use the one available ?
Is there a special depth you need for your test ?

Reply:
I did some tests with GRCh37 but we can remove these for now.